python-frontend/typeshed_serializer/runners/tox_runner.py

```diff
@@ -140,8 +141,17 @@ def main():
         logger.info("SKIPPING TYPESHED SERIALIZATION")
         # At the moment we need to run the tests in order to not break the quality gate.
         # If the tests are skipped this could potentially result in missing coverage.
+        if skip_tests:
+            logger.info("SKIPPING TYPESHED SERIALIZER TESTS")
+            return
         subprocess.run(['tox', '-e', 'py39'], check=True)
 
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--skip_tests')
```

Comment:
This could be simplified with `add_argument("--skip_tests", action="store_true")`. It will return a boolean so no need to check against a string. And the default will be `False`. Only if `--skip_test` is passed as an argument we will get `True`.